framework/math/SpatialDiscretization/CellMappings/Lagrange/LagrangeHexMapping.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CHITECH_LAGRANGEHEXMAPPING_H
+#define CHITECH_LAGRANGEHEXMAPPING_H
+
+#include "math/SpatialDiscretization/CellMappings/LagrangeBaseMapping.h"
+
+namespace chi_math::cell_mapping
+{
+
+/**Lagrange element mapping for a hexahedron.
+* \ingroup doc_CellMappings*/
+class LagrangeHexMapping : public LagrangeBaseMapping
+{
+public:
+  LagrangeHexMapping(const chi_mesh::MeshContinuum& grid,
+                     const chi_mesh::Cell& cell,
+                     const Quadrature& volume_quadrature,
+                     const Quadrature& surface_quadrature);
+
+protected:
+  double RefShape(uint32_t i, const Vec3& qpoint) const override;
```

Comment:
just for me, what does override to the right do again?

Reply:
The override states that this child class's method overrides the virtual method from the parent